File: uri_exercicios_em_C/begginer/as_abas_de_pericles.c
#include <stdio.h>
#include <string.h>

int main() {

    int n,m,i;

    scanf("%d %d",&n,&m);

    for(i = 0; i < m; i ++){
        char acao [7];
        scanf("%s",&acao);
        if(strcmp(acao,"fechou") == 0) n ++;
        else n --;
    }
    printf("%d\n", n);
    return 0;
}

File: beecrowd_exercicios_em_Java/src/beginner/AsAbasDePericles.java
package beginner;

import java.io.IOException;
import java.util.*;

public class AsAbasDePericles {
    static Scanner sc = new Scanner(System.in);

    public static void main(String[] args) throws IOException {
        Locale.setDefault(Locale.US);
        int n = sc.nextInt(), m = sc.nextInt();

        for(int i = 0; i < m; i ++){
            String acao = sc.next();

            if (acao.equals("fechou")) n++;
            if (acao.equals("clicou")) n--;

        }

        System.out.println(n);

    }

}